web/api/v2/minikit/user-grant-cycle/index.ts

```diff
@@ -0,0 +1,107 @@
+import { verifyApiKey } from "@/api/helpers/auth/verify-api-key";
+import { errorResponse } from "@/api/helpers/errors";
+import { validateRequestSchema } from "@/api/helpers/validate-request-schema";
+import { createSignedFetcher } from "aws-sigv4-fetch";
+import { NextRequest, NextResponse } from "next/server";
+import * as yup from "yup";
+
+const userGrantCycleQuerySchema = yup.object({
+  wallet_address: yup
+    .string()
+    .length(42)
+    .required("wallet_address is required"),
+  required_app_id: yup
```

Comment:
for consistency I believe other endpoints just use. The endpoint will error if it's required
```suggestion
  app_id: yup
```